nexus/src/app/project.rs

```diff
@@ -149,40 +147,4 @@ impl super::Nexus {
             .collect::<Result<Vec<_>, _>>()?;
         Ok(shared::Policy { role_assignments })
     }
-
-    pub(crate) async fn project_ip_pools_list(
-        &self,
-        opctx: &OpContext,
-        project_lookup: &lookup::Project<'_>,
-        pagparams: &PaginatedBy<'_>,
-    ) -> ListResultVec<db::model::IpPool> {
-        let (.., authz_project) =
-            project_lookup.lookup_for(authz::Action::ListChildren).await?;
-
-        self.db_datastore
-            .project_ip_pools_list(opctx, &authz_project, pagparams)
-            .await
-    }
-
-    pub fn project_ip_pool_lookup<'a>(
-        &'a self,
-        opctx: &'a OpContext,
-        pool: &'a NameOrId,
-        _project_lookup: &Option<lookup::Project<'_>>,
-    ) -> LookupResult<lookup::IpPool<'a>> {
-        // TODO(2148, 2056): check that the given project has access (if one
-        // is provided to the call) once that relation is implemented
-        match pool {
-            NameOrId::Name(name) => {
-                let pool = LookupPath::new(opctx, &self.db_datastore)
-                    .ip_pool_name(Name::ref_cast(name));
-                Ok(pool)
-            }
-            NameOrId::Id(id) => {
-                let pool =
-                    LookupPath::new(opctx, &self.db_datastore).ip_pool_id(*id);
-                Ok(pool)
-            }
-        }
-    }
```

Comment:
this was the same as regular ip_pool_lookup. deleted, use the latter instead. see `silo_ip_pool_fetch`